target/linux/ramips/image/mt7621.mk

```diff
@@ -1937,6 +1937,18 @@ define Device/snr_snr-cpe-me2-lite
 endef
 TARGET_DEVICES += snr_snr-cpe-me2-lite
 
+define Device/snr_cpe-me2-sfp
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := SNR
+  DEVICE_MODEL := SNR-CPE-ME2-SFP
+  UIMAGE_NAME := $$(DEVICE_MODEL)
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7663-firmware-ap \
```

Comment:
Why do you need kmod-mt7603, kmod-mt7615e, kmod-mt7663-firmware-ap? kmod-mt7615-firmware should be enough.